src/components/utils/popover/popover-abstract.ts

```diff
@@ -25,10 +26,18 @@ export abstract class PopoverAbstract<Nodes extends PopoverNodes = PopoverNodes>
    */
   protected nodes: Nodes;
 
+  /**
+   * List of usual interactive popover items that can be clicked, hovered, etc.
+   * (excluding separators)
+   */
+  protected get itemsInteractive(): PopoverItemDefault[] {
+    return this.items.filter(item => item instanceof PopoverItemDefault) as PopoverItemDefault[];
+  }
```

Comment:
Would be there any other interactive classes except for Default? Maybe add `isInteractive` getter to PopoverItem class, or check for `!(item instanceof PopoverItemSeparator)`

Reply:
For now there is only 1 interactive item type. If we add another, this condition will be updated